imagebuildah/chroot_symlink.go

```diff
@@ -24,7 +24,7 @@ func init() {
 func resolveChrootedSymlinks() {
 	status := 0
 	flag.Parse()
-	if len(flag.Args()) < 1 {
+	if len(flag.Args()) < 2 {
```

Comment:
Not sure why you made this change?  Might be good to add a comment in the description section of the PR.

Reply:
The description of the PR says why
`two arguments are required by the program`

Reply:
ALthough I think doing logrus.error would be helpful in this case before exiting.

Reply:
@giuseppe While you clean this up, could you add a logrus.Error message below.

Reply:
pushed a new version with an added error message